sw/ground_segment/misc/natnet2ivy.c

```diff
@@ -294,6 +294,14 @@ void natnet_parse(unsigned char *in) {
         memcpy(&rigidBodies[j].error, ptr, 4); ptr += 4;
         printf_natnet("Mean marker error: %3.8f\n", rigidBodies[j].error);
       }
+
+      // 2.6 and later
+      if( ((natnet_major == 2)&&(natnet_minor >= 6)) || (natnet_major > 2) || (natnet_major == 0) )
+      {
+          // params
+          short params = 0; memcpy(&params, ptr, 2); ptr += 2;
+          bool bTrackingValid = params & 0x01; // 0x01 : rigid body was successfully tracked in this frame
```

Comment:
Hm I just copied this from the SDK, the pointer increment is the important part. I can not find another use of the bTrackingValid boolean in their code. I can comment it out if you want.